### PR TITLE
highlight discussion time minimum

### PR DESCRIPTION
**Pull request by shelbeeee**

### CODE REVIEW

File: manual.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ Radworks manages proposals in **monthly cycles** to sync voting periods and esta
 | Week 3 | Formal Review | Third Monday |  Open proposal for Formal Review |
 | Week 4 | Submission | Fourth Monday | Submit proposal on-chain |
 
-**Proposals can be posted to the forum for Discussion at anytime throughout the monthly cycle, but requires at least 7 days on the forum to be considered for that month’s voting cycle. If proposal are posted after the second Monday deadline of a proposal cycle, they will be included in the next cycle. New proposals should be posted in the [Proposal Discussions](https://community.radworks.org/c/governance/discussions/19) sub-category under the Governance category on the forum.**
+> **Proposals can be posted to the forum for Discussion at anytime throughout the monthly cycle, but requires at least 7 days on the forum to be considered for that month’s voting cycle. If proposal are posted after the second Monday deadline of a proposal cycle, they will be included in the next cycle. New proposals should be posted in the [Proposal Discussions](https://community.radworks.org/c/governance/discussions/19) sub-category under the Governance category on the forum.**
```
Comment (abbey-titcomb) [inline manual.md]:
```suggestion
> **Proposals can be posted to the forum for Discussion at any time throughout the monthly cycle but require at least 7 days on the forum to be considered for that month's voting cycle. If proposals are posted after the second Monday deadline of a proposal cycle, they will be included in the next cycle. New proposals should be posted in the [Proposal Discussions](https://community.radworks.org/c/governance/discussions/19) sub-category under the Governance category on the forum.**
```

**Review by abbey-titcomb — CHANGES_REQUESTED** [review]
fixed some grammar

**Review by shelbeeee — COMMENTED** [review]
lgtm

**Review by abbey-titcomb — APPROVED** [review]
looks great!

**Pull request merged**